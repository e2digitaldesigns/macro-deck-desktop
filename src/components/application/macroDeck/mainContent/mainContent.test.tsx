import React from "react";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import TemplateMainContent from "./mainContent";

describe("<Template Header Component/>", () => {
  it("Should render without errors", () => {
    const { getByTestId } = render(<TemplateMainContent />);
    const component = getByTestId("template-main-content-component");
    expect(component).toBeTruthy();
  });

  it("Should render without errors", () => {
    const { getByTestId } = render(<TemplateMainContent />);
    const component = getByTestId("template-main-content-component");
    expect(component).toMatchSnapshot();
  });
});
