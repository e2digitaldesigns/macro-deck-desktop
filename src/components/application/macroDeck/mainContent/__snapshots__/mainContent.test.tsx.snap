// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Template Header Component/> Should render without errors 1`] = `
<div
  class="template-main-content"
  data-testid="template-main-content-component"
>
  <div
    data-testid="main-content-button-pads__null"
  />
  <div />
  <div />
</div>
`;
